cmd/cmd_internal_test.go

```diff
@@ -73,6 +74,11 @@ func TestCmdFlags(t *testing.T) {
 					Denylist:  "",
 					DBPath:    "",
 				},
+				Feature: featureset.Config{
```

Comment:
nit: add cmd test including enabled and disabled features too so that reader of this code would get more clarity with the use of these flags.
Because from the name of these flags it seems that they are boolean flags (like enabled=true/false).

Reply:
I'll see what I can do